infra/wca_on_rails/production/rails.tf

```diff
@@ -206,14 +206,14 @@ resource "aws_ecs_task_definition" "this" {
   task_role_arn      = aws_iam_role.task_role.arn
 
   cpu = "1024"
-  memory = "3954"
+  memory = "3910"
```

Comment:
Where does this change come from?

Reply:
the m6i.large instance reserves slightly more RAM for the ECS Agent then the m6i.2xlarge, so I had to tweak this number to fit in 2 container per instance.